Resources/Prototypes/Entities/Objects/Consumable/Smokeables/Cigarettes/cigarette.yml

```diff
@@ -14,6 +14,7 @@
   - type: Tag
     tags:
       - Cigarette
+      - Trash
```

Comment:
dont think cigarettes or joints need this if BaseSmokeable already has it

Reply:
Same as cola issue, the child won't inherit the tag if it has its own tag component.

This is just stop gap fix under the current tag system to make the trash bag more useful till maintainer meeting I guess.